core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature

```diff
@@ -0,0 +1,930 @@
+
+    # Check equity-like-shares before network moves forward
+    When the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 3000                    |
+      | lp2   | 0.9387755102040816 | 46000                   |
+    
+    # Trigger entry into next market period
+    And the network moves ahead "1" blocks:
+
+    # Confirm equity-like-shares are unchanged by the network moving forwards (as virtual-stakes scaled by same factor, r)
+    Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 3000                    |
+      | lp2   | 0.9387755102040816 | 46000                   |
+
+    # -------------------------------------------------------------------------------------------------------------------
+    # -------------------------------------------------------------------------------------------------------------------
+
+
+    # 2nd period: 1 LPs increase commitment, positive growth:
+    # -------------------------------------------------------------------------------------------------------------------
+
+    # Check equity-like-shares before liquidity amendment
+    Given the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 3000                    |
+      | lp2   | 0.9387755102040816 | 46000                   |
+
+    When the parties submit the following liquidity provision:
+      | id  | party | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type   |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | buy  | BID              | 1          | 2      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | buy  | MID              | 3          | 1      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | sell | ASK              | 1          | 2      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | sell | MID              | 3          | 1      | amendment |
+
+    # Confirm equity-like-shares updated immediately after liquidity amendment
+    Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share    | average entry valuation           |
+      | lp1   | 0.0794314813996635   | 3971.5740699831772119860611436306 |
+      | lp2   | 0.9205685186003365   | 46000                             |
+
+    # ERROR:
+    # - "equity-like-share" values calculated incorrectly.
+    # - "average-entry-valuation" values calculated incorrectly, sum of "average-entry-valuation" values does not match sum of "physical-stake" values.
```

Comment:
I don't see how we'd expect the sum of the average entry valuation to match the sum of the physical stake

Reply:
Hey, yep following the updated `average-entry-valuation` in spec 0042 those comments in the test don't make sense any more. I'll remove the comments from the feature test and update all the values.